corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -17,11 +17,65 @@ hqDefine("geospatial/js/case_grouping_map",[
         self.coordinates = coordiantes;
         self.caseLink = caseLink;
 
+        // TODO: Group ID needs to be set
+        self.groupId = null;
+
         return self;
     }
 
+    function exportModel() {
+        var self = {};
+
+        self.casesToExport = ko.observableArray([]);
+
+        self.handleExportCSV = function () {
+            const casesToExport = _.map(self.casesToExport(), function (caseItem) {
+                const coordinates = (caseItem.coordinates) ? `${caseItem.coordinates.lng} ${caseItem.coordinates.lat}` : "";
+                return {
+                    'groupId': caseItem.groupId,
+                    'caseId': caseItem.caseId,
+                    'coordinates': coordinates,
+                };
+            });
+            exportToCsv(casesToExport);
+        };
+
+        function exportToCsv(itemsArr, includeHeaders = true) {
+            if (!itemsArr.length) {
+                return;
+            }
+
+            let csvStr = "";
+            if (includeHeaders) {
```

Comment:
when would this be false?

Reply:
It will currently always be `true`. The reason I put it in was to provide an easy way to remove these headers should we ever want to change the functionality in the future for only downloading the data. Since it's not currently being used, I'm happy to remove this if needed.

Reply:
YAGNI

Additionally, I don't believe a file without headers will make sense for the user.

Reply:
I have removed the unnecessary parameter, and combined the two export functions together since we don't require any input parameters. ([1ba4d8b](https://github.com/dimagi/commcare-hq/pull/33610/commits/1ba4d8b0f9cbcd45cd62a47c06201059cf159a1e))